<launch>

    <!-- Launch teleop xbox360 twist -->
    <arg name="twist_topic" default="twist_controller/command"/>
    <arg name="arm_group" default="arm"/>
    <arg name="frame_id" default="base_link"/>
    <arg name="end_effector" default="tcp_link"/>

    <node pkg="morpheus_teleop" type="xbox360_swap.py" name="xbox360_twist" output="screen">
        <param name="twist_topic" value="$(arg twist_topic)"/>
        <param name="arm_group" value="$(arg arm_group)"/>
    </node>

  <!-- Launch Twist converter-->
  <include file="$(find morpheus_teleop)/launch/twist_converter.launch">
      <arg name="twist_topic" value="$(arg twist_topic)"/>
      <arg name="frame_id" value="$(arg frame_id)"/>
      <arg name="end_effector" value="$(arg end_effector)"/>
  </include>

</launch>